src/Container/ServiceManager/AutowireFactory.php

```diff
@@ -26,9 +26,8 @@ public function __construct(?GenericAutowireFactory $factory = null)
      * Check creatability of the requested name
      *
      * @param string $requestedName
-     * @return bool
      */
-    public function canCreate(ContainerInterface $container, $requestedName)
+    public function canCreate(ContainerInterface $container, $requestedName): bool
```

Comment:
Right, in this case, `laminas-service-manager` `4.0` cannot be supported without a major release.

In fact we have to add a conflict to `laminas-service-manager >= 4.0` for the `3.x` branches, since this class is not interface compatible with that version.

Another question: Is this even done correctly by editing this branch or should the renovate Issues rather be addressed with a separate PR?

Reply:
Already changed the milestone here. I have some suggestions for `4.0.x` in #95 to be addressed with the TSC.
This component needs some cleanup 😬